common/views/components/Body/Body.js

```diff
@@ -39,6 +38,7 @@ const Body = ({ body }: Props) => {
         .map((slice, i) =>
           <div className={classNames({
             'body-part': true,
+            [spacing({s: 3}, {padding: ['top']})]: i === 0 && slice.type !== 'imageGallery',
```

Comment:
Could we not wrap the image gallery in a spacer keep it closer to the metal?

Reply:
The issue is with needing to remove the padding from the top if the image gallery follows the header – didn't feel great about this, but didn't see how else to do it

Reply:
Isn't "follows the header" = `i === 0` which we can do anywhere in the loop?